cli-commands/commands.js

```diff
@@ -61,6 +62,20 @@ const commands = [
 		commandProcessor: (argv) => {
 			history.run(argv.limit, console);
 		}
+	},
+	{
+		command: 'compile [dir]',
+		description: 'Compiles the smart contracts that are in the directory contracts in the path provided by the dir parameter (defaults to .)',
+		argumentsProcessor: (yargs) => {
+			yargs.positional('dir', {
+				describe: 'Specifies the default dir to all folders',
```

Comment:
Lets write this as `specifies the root dir to read the contracts and place the build folder`